cfme/tests/configure/test_ntp_server.py

```diff
@@ -0,0 +1,89 @@
+import fauxfactory
+from cfme import login
+from cfme.configure import configuration
+from datetime import datetime
+from fixtures.pytest_store import store
+from utils.conf import cfme_data
+from utils.log import logger
+from utils.ssh import SSHClient
+from utils.wait import wait_for
+
+
+def ssh_command_execution(command):
```

Comment:
Seeing as the appliance already has an SSH object, it probably makes most sense here to just use that object and do away with this function completely.

```status, msg = store.current_appliance.ssh_client.run_command(..............)``` 

as this will reuse the existing ssh connection from the client store, making your code considerably faster.